ratis-server/src/main/java/org/apache/ratis/server/metrics/SegmentedRaftLogMetrics.java

```diff
@@ -67,6 +67,10 @@
   /** Number of entries appended to the raft log */
   public static final String RAFT_LOG_APPEND_ENTRY_COUNT = "appendEntryCount";
   public static final String RAFT_LOG_PURGE_METRIC = "purgeLog";
+  /** Time taken for a Raft log operation to complete write execution. */
+  public static final String RAFT_LOG_WRITE_STATEMACHINE_TIMEOUT_COUNT = "numStateMachineWriteTimeout";
+  /** Time taken for a Raft log operation to complete read execution. */
+  public static final String RAFT_LOG_READ_STATEMACHINE_TIMEOUT_COUNT = "numStateMachineReadTimeout";
```

Comment:
Let's call them "numStateMachineDataWriteTimeout" and "numStateMachineDataReadTimeout".
```
  /** Time taken for a Raft log operation to complete write state machine data. */
  public static final String RAFT_LOG_STATEMACHINE_DATA_WRITE_TIMEOUT_COUNT = "numStateMachineDataWriteTimeout";
  /** Time taken for a Raft log operation to complete read state machine data. */
  public static final String RAFT_LOG_STATEMACHINE_DATA_READ_TIMEOUT_COUNT = "numStateMachineDataReadTimeout";
```